tests/ts_updown_util.erl

```diff
@@ -341,33 +341,35 @@ fmt(F, A) ->
 
 make_tables(#test_set{create = #create{should_skip = true}}, _TableNode) ->
     pass;
-make_tables(#test_set{timestamp = Timestamp,
-                      create    = #create{ddl      = DDLFmt,
+make_tables(#test_set{testname  = Testname,
+                      timestamp = Timestamp, 
+                      create    = #create{ddl      = DDLFmt, 
                                           expected = Exp}}, TableNode) ->
     %% fast machines:
     timer:sleep(1),
-    Table = get_table_name(Timestamp),
+    Table = get_table_name(Testname, Timestamp),
     DDL = fmt(DDLFmt, [Table]),
     Client1 = rt:pbc(TableNode),
     case riakc_ts:'query'(Client1, DDL) of
         Exp ->
             ok = wait_until_active_table(Client1, Table, 5),
             ct:log("Table ~p created on ~p", [Table, TableNode]),
             pass;
-        Error ->
-            ct:log("Failed to create table ~p: (~s)", [Table, Error]),
+        {error, {_No, Error}} ->
```

Comment:
Are we certain the error messages from query are always going to be in the form of a line number tuple? The specs in `riak-erlang-client` do not "enforce" that.